crates/nu-protocol/src/value/mod.rs

```diff
@@ -1462,10 +1425,9 @@ impl Value {
                     },
                 }
             }
-            _ => {
-                let path_member = cell_path.first().expect("there is a first");
+            [member, path @ ..] => {
```

Comment:
Chef's kiss! Love how this is both explicit and gets rid of a bunch of triple checking

Reply:
Pattern matching is too powerful! :smile: